proto/minder/v1/minder.proto

```diff
@@ -1993,10 +2059,18 @@ enum RuleTypeReleasePhase {
 // The version is assumed from the folder's version.
 message RuleType {
     // version is the version of the rule type API.
-    string version = 11;
+    string version = 11 [
+        (buf.validate.field).string = {
+            pattern: "^v\\d+(\\.\\d+)?(\\.\\d+)?(?:-[\\w\\.-]+)?$",
```

Comment:
Again, I'd prefer just `v1` than `v1.1.7` here for now.

Reply:
Fixed 👍